src/three-components/ModelScene.ts

```diff
@@ -107,6 +102,8 @@ export default class ModelScene extends Scene {
     this.pivot = new Object3D();
     this.pivot.name = 'Pivot';
 
+    this.skyboxMesh = this.createSkyboxMesh();
```

Comment:
Honestly the mesh is so insignificant, I wouldn't bother. We still need to cache modelScene properly anyway, since it's still leaking shadow textures and such. I think this can fit in nicely there.